lib/lighthouse-reporter.js

```diff
@@ -93,7 +93,40 @@ async function writeReport(
     categoryReport[category.id] = Math.round(category.score * 100);
   }
 
-  return { categoryReport, htmlReport };
+  const auditReport = {};
+
+  for (const auditName in lighouseResult.lhr.audits) {
+    if (
+      !Object.prototype.hasOwnProperty.call(
+        lighouseResult.lhr.audits,
+        auditName,
+      )
+      || lighouseResult.lhr.audits[auditName].scoreDisplayMode === "notApplicable"
+      || lighouseResult.lhr.audits[auditName].scoreDisplayMode === "manual"
```

Comment:
store these results outside too so it's more clear what's going on

Reply:
Reorganized it a bit.